dataplane/vppagent/scripts/vpp_monitor_handlers.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+
+def should_save_core_dump():
+    return os.environ.get('SAVE_COREDUMP') == 'true'
+
+
+def core_dump_location():
+    return os.environ.get('POSTMORTEM_DATA_LOCATION', '/var/tmp/nsm-postmortem/vpp-dataplane')
+
+
+def save_stacktrace():
+    gdb.execute("cd /tmp")
+    gdb.execute("set logging file vpp_backtrace")
+    gdb.execute("set logging on")
+    gdb.execute("thread apply all bt")
+    gdb.execute("set logging off")
+
+
+def save_core_dump():
+    gdb.execute("cd " + core_dump_location())
+    gdb.execute("generate-core-file")
```

Comment:
Could we compress core files using gzip2? Should they be smaller in this case?

Reply:
Good idea :)  Corefiles are *huge*